1080i/Includes_Images.xml

```diff
@@ -4,20 +4,24 @@
     <variable name="PercentWatched">
         <value condition="Skin.HasSetting(hide.markers)"></value>
         <value condition="ListItem.IsPlaying">indicator/play.png</value>
+        <value condition="ListItem.IsResumable + Skin.HasSetting(hide.markers.inprogress)"></value>
```

Comment:
We need to match if we want to return empty, otherwise could display unwatched or new flags...